LICENSE.md

```diff
@@ -1025,48 +1025,45 @@ Section 6 -- Term and Termination.
      right the Licensor may have to seek remedies for Your violations
      of this Public License.
 
-  c. For the avoidance of doubt, the Licensor may also offer the
-     Licensed Material under separate terms or conditions or stop
-     distributing the Licensed Material at any time; however, doing so
-     will not terminate this Public License.
-
-  d. Sections 1, 5, 6, 7, and 8 survive termination of this Public
-     License.
+c. For the avoidance of doubt, the Licensor may also offer the
+Licensed Material under separate terms or conditions or stop
+distributing the Licensed Material at any time; however, doing so
+will not terminate this Public License.
 
+d. Sections 1, 5, 6, 7, and 8 survive termination of this Public
+License.
 
 Section 7 -- Other Terms and Conditions.
 
-  a. The Licensor shall not be bound by any additional or different
-     terms or conditions communicated by You unless expressly agreed.
-
```

Comment:
IMHO, this blank line should have been removed

Reply:
I'm not sure what you mean.
There is a blank line between all subsections in every section, why remove it here?

Reply:
Sorry - that was a reviewer mistake. My bad. Closing this as resolved.